cabot/dummyapp/migrations/0002_dummy_check.py

```diff
@@ -1,110 +0,0 @@
-            'name': ('django.db.models.fields.TextField', [], {}),
-            'polymorphic_ctype': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "u'polymorphic_cabotapp.statuscheck_set'", 'null': 'True', 'to': u"orm['contenttypes.ContentType']"})
-        },
-        u'contenttypes.contenttype': {
-            'Meta': {'ordering': "('name',)", 'unique_together': "(('app_label', 'model'),)", 'object_name': 'ContentType', 'db_table': "'django_content_type'"},
-            'app_label': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
-            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'model': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
-            'name': ('django.db.models.fields.CharField', [], {'max_length': '100'})
-        },
-        'dummyapp.dummysource': {
-            'Meta': {'object_name': 'DummySource', '_ormbases': ['metricsapp.MetricsSourceBase']},
-            u'metricssourcebase_ptr': ('django.db.models.fields.related.OneToOneField', [], {'to': "orm['metricsapp.MetricsSourceBase']", 'unique': 'True', 'primary_key': 'True'})
-        },
-        'dummyapp.dummystatuscheck': {
-            'Meta': {'ordering': "['name']", 'object_name': 'DummyStatusCheck', '_ormbases': ['metricsapp.MetricsStatusCheckBase']},
-            'counter': ('django.db.models.fields.IntegerField', [], {'default': '0'}),
-            u'metricsstatuscheckbase_ptr': ('django.db.models.fields.related.OneToOneField', [], {'to': "orm['metricsapp.MetricsStatusCheckBase']", 'unique': 'True', 'primary_key': 'True'})
-        },
-        'metricsapp.metricssourcebase': {
-            'Meta': {'object_name': 'MetricsSourceBase'},
-            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'name': ('django.db.models.fields.CharField', [], {'unique': 'True', 'max_length': '30'})
-        },
-        'metricsapp.metricsstatuscheckbase': {
-            'Meta': {'ordering': "['name']", 'object_name': 'MetricsStatusCheckBase', '_ormbases': [u'cabotapp.StatusCheck']},
-            'check_type': ('django.db.models.fields.CharField', [], {'max_length': '30'}),
-            'high_alert_importance': ('django.db.models.fields.CharField', [], {'default': "'ERROR'", 'max_length': '30'}),
-            'high_alert_value': ('django.db.models.fields.FloatField', [], {'null': 'True'}),
-            'source': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['metricsapp.MetricsSourceBase']"}),
-            u'statuscheck_ptr': ('django.db.models.fields.related.OneToOneField', [], {'to': u"orm['cabotapp.StatusCheck']", 'unique': 'True', 'primary_key': 'True'}),
-            'time_range': ('django.db.models.fields.IntegerField', [], {'default': '30'}),
-            'warning_value': ('django.db.models.fields.FloatField', [], {'null': 'True'})
-        }
-    }
-
-    complete_apps = ['dummyapp']
-    symmetrical = True
```

Comment:
This isn't actually going to delete the existing dummy check...we'd need another data migration for that (or we can just delete it manually since it's one thing that doesn't really do anything)